build-rev.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# This command is used by bazel as the workspace_status_command
+# to implement build stamping with git information.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Do not use plumbing commands, like git diff-index, in this target. Our build
+# process modifies files quickly enough that plumbing commands report false
+# positives on filesystems with only one second of resolution as a performance
+# optimization. Porcelain commands, like git diff, exist to detect and remove
+# these false positives.
+#
+# For details, see the "Possible timestamp problems with diff-files?" thread on
+# the Git mailing list (http://marc.info/?l=git&m=131687596307197).
+
+BUILDTYPE="development"
+LDFLAGS=""
+
+GIT_BUILD_TYPE=(${BUILDTYPE} -extldflags "${LDFLAGS}")
```

Comment:
? Why have we done this? Is there a reason the build type isn't just `development`?

Reply:
Actually I just copied what is there in `Makefile` but I think I can leave it only as `development`

Reply:
That line sets the `LINKFLAGS` variable. It's not all part of the `BUILDTYPE` thing.

Reply:
Let me remove it then and just put `development` in the actual binary generated by `Makefile` it is showing only `development `